vars/main.yml

```diff
@@ -2,3 +2,4 @@
 es_package_url: "https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch"
 reserved_xpack_users: ["elastic","kibana","logstash_system"]
 sysd_config_file: "/etc/systemd/system/elasticsearch.service.d/override.conf"
+sysd_always_restart: True
```

Comment:
I think this should be a `default` and not a `var` if we want it to be overridable (maybe there are cases in which we don't want ES to autorestart? I'm not sure). The wording you used in the README makes it sound like you want it to be configurable. ^^

Reply:
You're rigth! Fixed!